pkg/filter/filter.go

```diff
@@ -32,7 +32,7 @@ type Filter interface {
 	// ShouldDiscardDDL returns true if this DDL should be discarded.
 	// If a ddl is discarded, it will neither be applied to cdc's schema storage
 	// nor sent to downstream.
-	ShouldDiscardDDL(ddlType timodel.ActionType) bool
+	ShouldDiscardDDL(ddlType timodel.ActionType, schema, table string) bool
```

Comment:
`ShouldDiscardDDL` and `ShouldIgnoreDDLEvent` confuses me.
What's the difference?

Reply:
`Discard` means we discard a DDL in puller, a discarded DDL will not enter other logic of TiCDC, `Ignore` means we ignore a DDL before we sink it, a ignored DDL will be applied to `schemaStorage`.

Reply:
Seems that in logically, they only have different input params.
The whole ticdc don't care about the the ignored or discarded DDL.